readme.md

```diff
@@ -131,6 +131,12 @@ Default: `0`
 
 Priority of operation. Operations with greater priority will be scheduled first.
 
+##### any other key
+
+Type: `object`
+
+You can pass any keys in options to use then later in sizeBy filtering
```

Comment:
There should rather be a description under `#### options`, that you can pass any other key.